### PR TITLE
EDGE-170 Add API call to return Connect SDK version

### PR DESCRIPTION
**Pull request by ionyshch-prismsl**
Added prism::connect::getSdkVersion()
Return SDK version that has major, minor and revision components.

Updated CMakeLists.txt and created build.sh to create delivery
on demand OR when build on teamcity from annotated tag.

### CODE REVIEW

File: include/client.h [hunk 1]
```diff
@@ -11,6 +11,35 @@ namespace prism
...
+    uint32_t toUint() const
+    {
+        return ((uint32_t)major << 24) + ((uint32_t)minor << 16) + ((uint32_t)revision << 8);
+    }
+
+    std::string toString() const;
+
+    uint8_t major;
+    uint8_t minor;
+    uint8_t revision;
```
Comment (vsharapov-prism) [inline include/client.h:37]: uint8 is enough? I came across situation that 8 bit is too small

Comment (ionyshch-prismsl) [reply include/client.h:37]: major is likely be small number as 1, 2 or 3 (even firefox/google chrome versions are below 100)
minor should be enough
revision starts with 0 for each fixed major and minor to identify changes, that doesn't change API e.g. fixes, refactoring.

Comment (vsharapov-prism) [reply include/client.h:37]: I understand in this case should be enough

**Review by vsharapov-prism — APPROVED** [review]

File: include/client.h [hunk 2]
```diff
@@ -11,6 +11,35 @@ namespace prism
 namespace connect
 {
 
+struct SdkVersion
+{
+    // ugly workaround for ugly bug in gcc, defining macros major() and minor()
+#undef major
+#undef minor
+
+    SdkVersion(uint8_t major, uint8_t minor, uint8_t revision)
```
Comment (Yurmusp) [inline include/client.h:20]: I would say revision as 8 bit int is not enough. We may have much more than 255. I would make revision 16 bit uint

**Pull request merged**